letsencrypt/cli.py

```diff
@@ -1040,8 +1040,7 @@ def main(cli_args=sys.argv[1:]):
 
     # note: arg parser internally handles --help (and exits afterwards)
     plugins = plugins_disco.PluginsRegistry.find_all()
-    parser, tweaked_cli_args = create_parser(plugins, cli_args)
-    args = parser.parse_args(tweaked_cli_args)
```

Comment:
The variable `tweaked_cli_args` was used to preserve the original value of `args` because something about the modified version caused problems for configuration.NamespaceConfig.  I can't remember what it was, though.  Unsure if you'll need to reintroduce that preservation.